src/components/TransactionList/TransactionList.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+
+// Example list of transactions
+// TODO: to be removed and saved into localStorage
```

Comment:
when introducing localStorage you will still need this data in you project, you will just replace a useState with a useLocalStorageState and then need initial data as well.